### DISCUSSION TITLE
Build vs. Resources

### DISCUSSION THREAD

**Discussion by BrendanC23** [alex8088/electron-vite#394] [Q&A]
The [react-ts boilerplate](https://github.com/alex8088/quick-start/tree/master/packages/create-electron/playground/react-ts) has both a `build` and a `resources` directory. Both of these have an `icon.png` file. Based on the electron-builder [documentation](https://www.electron.build/icons.html), it looks in `/build` for the icon to use.

What is the purpose of having both build and resources folders? If I want to edit the icon for my app, do I need to update the icon in both locations?

**Comment by alex8088** [comment]
The build dir is for electron-builder and will not be distributed. The resources dir for your app.

**Comment by Ryanjso** [comment]
This is also unclear to me. I know the icons in the build folder are used by electron-builder to set the icon, but what is the purpose of importing an icon from the resource file into the `createWindow` function? It doesn't seem to do anything for me on mac. 

Additionally, it's unclear to me how to change the app icon in a development build.
